docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '1.0'
+
+services:
+  redis:
+    image: redis-cache:latest
+    build: ./redis
+    ports:
+      - '6379:6379'
+    logging:
+      options:
+        max-size: '10m'
+        max-file: '5'
+    volumes:
+      - redis-data:/data
+  WZDx_tim_translator:
+    build: ./Translators/WZDx
+    image: translator:latest
+    environment:
+      - dual_carriageway_endpoint=${dual_carriageway_endpoint}
+      - DB_HOST=${DB_HOST}
+      - DB_NAME=${DB_NAME}
+      - DB_PASS=${DB_PASS}
+      - DB_USER=${DB_USER}
+      - LOGGING_LEVEL=${LOGGING_LEVEL}
+      - WZDX_ENDPOINT=${WZDX_ENDPOINT}
+      - WZDX_API_KEY=${WZDX_API_KEY}
+      - REDIS_HOST=${REDIS_HOST}
```

Comment:
Do these "REDIS_XXX" values point to the above redis service?

Reply:
These values do point to the above redis service